### DISCUSSION TITLE
Cannot update "reuse private key on renewal setting"

### DISCUSSION THREAD

**Discussion by dazinator** [polymind-inc/acmebot#567] [Q&A] [answered]
Hi,

I am unable to amend the "reuse private key on renewal setting" for a certificate.

Is there some way to do this?

![image](https://user-images.githubusercontent.com/3176632/222393563-29aa89a0-e058-402d-9ed1-3e0bdce1e16d.png)

**Comment by shibayan** [comment, marked as answer]
This screen is read-only. If changes are necessary, please edit the Issuance Policy individually from the Key Vault.

![image](https://user-images.githubusercontent.com/1356444/226659014-8f2f6223-19c3-4925-ba76-33ad913ce0fc.png)

**Reply by dazinator** [reply]
Ah thank you.
